chef/cookbooks/ceph/recipes/calamari.rb

```diff
@@ -65,3 +65,82 @@
 # but is not critical given the user is prompted to authorize them the first
 # time they log in to Calamari.
 
+include_recipe 'apache2'
+
+use_ssl = node['ceph']['calamari']['ssl']['enabled']
+if use_ssl
+  include_recipe 'apache2::mod_ssl'
+
+  certfile      = node['ceph']['calamari']['ssl']['certfile']
+  keyfile       = node['ceph']['calamari']['ssl']['keyfile']
+  if node['ceph']['calamari']['ssl']['generate_certs']
+    package "openssl"
+    ruby_block "generate_certs for calamari" do
```

Comment:
See https://build.opensuse.org/package/view_file/Cloud:OpenStack:Master/openstack-dashboard/openstack-dashboard.spec?expand=1 and https://build.opensuse.org/package/view_file/Cloud:OpenStack:Master/openstack-suse/macros.openstack?expand=1